qualtran/_infra/gate_with_registers.py

```diff
@@ -149,16 +148,37 @@ class GateWithRegisters(Bloq, cirq.Gate, metaclass=abc.ABCMeta):
     def decompose_bloq(self) -> 'CompositeBloq':
         """Decompose this Bloq into its constituent parts contained in a CompositeBloq.
 
-        Bloq users can call this function to delve into the definition of a Bloq. If you're
-        trying to define a bloq's decomposition, consider overriding `build_composite_bloq`
-        which provides helpful arguments for implementers.
+        Bloq users can call this function to delve into the definition of a Bloq. The function
+        returns the decomposition of this Bloq represented as an explicit compute graph wrapped
+        in a `CompositeBloq` object.
+
+        Bloq authors can the bloq's decomposition by overriding any of the two convenience methods:
```

Comment:
"Bloq authors can the"

Reply:
I'd personally remove "convenience" and just call them methods. A convenience method in my book is a method that removes a few keystrokes from calling it but under the hood just delegates to the full-featured thing maybe with some more aggressive defaults

Reply:
done